packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -137,6 +137,16 @@ export interface IDeltaHandler {
 		localOpMetadata: unknown,
 	) => void;
 
+	/**
+	 * Process messages for this channel. The messages here are contiguous messages for this channel in a batch.
+	 * @param messagesWithMetadata - A bunch of contiguous messages for this channel and their metadata.
+	 * @param local - true if the messages were originally generated by the client receiving it.
+	 */
+	processMessages?: (
```

Comment:
What does it mean for this to be `undefined`? Would probably be worth documenting the default behavior around this.

Reply:
made it required. Its all internal even though the type says alpha / legacy. This should not break partners.